python/GafferTest/HiddenFilePathFilterTest.py

```diff
@@ -45,17 +46,17 @@ class HiddenFilePathFilterTest( GafferTest.TestCase ) :
 
 	def test( self ) :
 
-		hiddenFile = Gaffer.FileSystemPath( os.path.join( self.temporaryDirectory(), ".sneaky.txt" ) )
+		hiddenFile = Gaffer.FileSystemPath( self.temporaryDirectory().joinpath( ".sneaky.txt" ).as_posix() )
 		with open( hiddenFile.nativeString(), "w" ) as f :
 			f.write( "Can't see me" )
 		if os.name == "nt" :
 			subprocess.check_call( [ "attrib", "+H", hiddenFile.nativeString() ] )
 
-		visibleFile = Gaffer.FileSystemPath( os.path.join( self.temporaryDirectory(), "frank.txt" ) )
+		visibleFile = Gaffer.FileSystemPath( self.temporaryDirectory().joinpath( "frank.txt" ).as_posix() )
```

Comment:
Same note as above

Reply:
Also fixed in https://github.com/GafferHQ/gaffer/commit/cac9d6fe99d912b396c2bbcd069d9f550c82869c.